src/Microsoft.IIS.WebManager/index.html.versioned

```diff
@@ -0,0 +1,38 @@
+﻿<!DOCTYPE html>
```

Comment:
Good idea. This could even be used for other transformations if ever necessary. I would probably have named it index.html.in since it doesn't necessarily have to be only for versions.

Reply:
changed